aiida_cp2k/workchains/aiida_base_restart.py

```diff
@@ -303,6 +255,40 @@ def _handle_calculation_sanity_checks(self, calculation):
         :param calculation: the calculation whose outputs should be checked for consistency
         :return: `ErrorHandlerReport` if a new calculation should be launched or abort if it includes an exit code
         """
+        import importlib
+        is_handled = False # Becomes True if at least one error handling operation was performed and was sucessfull
+        not_handled = False # Becomes True if at least one error handling operation was performed and was NOT sucessfull
+
+        for fxtr in sorted(self.inputs.fixers.keys()):
+            module_path, fixer_name, *args = self.inputs.fixers[fxtr]
+            module = importlib.import_module(module_path)
```

Comment:
what you are doing here is exactly what entry points were designed to achieve

Reply:
until we have a "FixerFactory", you can probably just put your fixers into a group `cp2k.fixers` and then load them by name
```
pgk_resources.load_entry_point(dist='aiida-cp2k', group='cp2k.fixers', name=name)
```
This way you don't have to pass the module path etc.

Reply:
Hmm, maybe you are right.

My intention was to have them loadable from any place accessible for python. So if one wants to add a fixer, one can just say:

```
{
    "fixer_xxx": ('from_where_to_import', 'what_to_import')
}
```

without a need for adding more stuff.

Reply:
@ltalirz  do you think it is worth to add the entry point mechanism here?

Reply:
I think so, because I believe this is where this will be going anyhow, and switching the name of the entry point group to `aiida.fixers` (or whatever it will be) will be easier than having to change the format for how to specify a fixer.